### PR TITLE
added setting to hide menu items, moved file manager to top-level

### PR DESCRIPTION
**Pull request by eadmaster**
Added:
-  setting to disable menu items, moved file manager to top-level https://github.com/pr3y/Bruce/issues/326

Currently this setting is unexposed in the "Config" menu, it can be used by editing `bruce.conf` manually or via serial cmds:

````
set disabledMenus Connect
set disabledMenus FM
set disabledMenus NRF24
````

Will add this setting array:
````
{
....
  "disabledMenus": [
    "Connect",
    "FM",
    "NRF24"
  ]
}
````

Fixes:

 - prioritize SD over LittleFS in some serialcmds
 - disabled GIF support on non-Cardputer devices (not working/freezing)

### CODE REVIEW

File: src/modules/ir/ir_read.cpp
Comment (bmorcelli) [inline src/modules/ir/ir_read.cpp:1]: This warning is causing too much confusion.. is better keep it as a serial print IMO..

Comment (eadmaster) [reply src/modules/ir/ir_read.cpp:1]: same with rf then...
https://github.com/pr3y/Bruce/blob/main/src/modules/rf/rf.cpp#L759

Comment (bmorcelli) [reply src/modules/ir/ir_read.cpp:1]: Yes, I haven't seen that warning.. but it not necessary too.. on RF I focused on RF_scan_copy function, that is usually called by the menu.. this function is called by serialcmds.. you can comment it ☺️

**Comment by eadmaster** [comment]
Restored the file manager entries in the "Others" menu, so if you don't like the new "Files" top-level menu , you can simply disable it and use the old entries.

**Comment by Teapot174** [comment]
Add this!!! It should be in Bruce!!!

**Comment by Tywors** [comment]
Where is the file 'bruce.conf' ?

**Pull request merged**